client/src/assets/scss/_new.scss

```diff
@@ -4631,27 +4631,115 @@ div.page-activity.messaging-center {
   .resilience-message {
     margin-bottom: 30px;
 
-    .title,
-    .title-wrapper {
+    .title-wrapper, .header {
       background-color: $color-velvet;
       color: $color-white;
+
+    }
+    .header {
+      display: flex;
+      flex-flow: row wrap;
       padding: 20px;
+      &.blue {
+        background-color: $color-primary;
+      }
+      &.purple {
+        background-color: $color-purple;
+      }
     }
 
     .title-wrapper {
       display: flex;
       justify-content: space-between;
     }
 
-    .title-wrapper .title {
+    .icon-options {
+      margin-top: 20px;
+      padding: 20px;
+      background: url(#{$img-path}icon-options.svg) center center no-repeat;
+      background-size: 32px 30px;
+      height: 32px;
+      width: 30px;
+      }
+
+    .category-header{
+      font-weight: bold;
+      font-size: 24px;
+      flex: 1;
+    }
+
+    .header .title {
       padding: 0;
     }
 
+    .date-sent {
+      flex-basis: 100%;
+      width: 0;
```

Comment:
Why is this needed?

Reply:
> Why is this needed?

I used flex-basis for the date to come on the new line line